docs/admin/auth/hba.rst

```diff
@@ -228,14 +228,6 @@ this:
             ssl: on
             method: cert
 
-Here, communication between nodes in the same zone (i.e., matching the
```

Comment:
I'd also change the sentence above from

```
For example, if you wanted to configure a `multi-zone cluster`_, you could do
this:
```

to 

```
For example, if you wanted to configure a `multi-zone cluster`_, you should enable certificate authentication like 
this:
```

and just have a single entry:


```
    auth:
      host_based:
        enabled: true
        config:
          0:
            protocol: transport
            ssl: on
            method: cert
```